queries/tags.scm

```diff
@@ -0,0 +1,33 @@
+(class_declaration
+ name: [
+        (identifier) @name
+        ]
+ ) @definition.class
+
+(interface_declaration
+ name: [
+        (identifier) @name
+        ]
+ ) @definition.interface
+
+(method_declaration
+ name: [
```

Comment:
I think it would be nice to format these consistently with other tags.scm - just wrt indentation and spacing.

Reply:
Heh, yeah, this is what Emacs gave us by default. Its indentation rules are… interesting.